DigitalLearningSolutions.Web/appsettings.json

```diff
@@ -49,5 +49,23 @@
     "ProductId": "80000003097"
   },
   "ShowAlertBanner": "True",
-  "AlertBannerContent": "<p class='nhsuk-body-s'>Creating a new NHS England: Health Education England, NHS Digital and NHS England have merged. <a href='https://hee.nhs.uk/hee-merges-with-nhs-england' target='_parent'>Learn more.</a></p>"
+  "AlertBannerContent": "<p class='nhsuk-body-s'>Creating a new NHS England: Health Education England, NHS Digital and NHS England have merged. <a href='https://hee.nhs.uk/hee-merges-with-nhs-england' target='_parent'>Learn more.</a></p>",
+  "IpRateLimiting": {
+    "EnableEndpointRateLimiting": true,
+    "StackBlockedRequests": false,
+    "RealIpHeader": "X-Real-IP",
+    "HttpStatusCode": 429,
+    "GeneralRules": [
+      {
+        "Endpoint": "post:/ForgotPassword",
+        "Period": "30s",
+        "Limit": 1
+      },
+      {
+        "Endpoint": "post:/Login",
+        "Period": "30s",
+        "Limit": 1

```

Comment:
I'm thinking that we would want to allow more than 1 login attempt in 30 seconds. 5 in one minute might be better?